include/OnnxMlirCompiler.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===------- OnnxMlirCompiler.h - ONNX-MLIR Compiler API Declarations -----===//
+//
+// This file contains declaration of onnx-mlir compiler functionality
+// exported from the OnnxMlirCompiler library
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef ONNX_MLIR_ONNXMLIRCOMPILER_H
+#define ONNX_MLIR_ONNXMLIRCOMPILER_H
+
+#include <onnx-mlir/Compiler/OMCompilerTypes.h>
+
+#ifdef ONNX_MLIR_BUILT_AS_STATIC
+#define ONNX_MLIR_EXPORT
+#define ONNX_MLIR_NO_EXPORT
+#else
+#ifdef OnnxMlirCompiler_EXPORTS
+/* We are building this library */
+#define ONNX_MLIR_EXPORT __declspec(dllexport)
+#else
+/* We are using this library */
+#define ONNX_MLIR_EXPORT __declspec(dllimport)
+#endif
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+namespace onnx_mlir {
+
+/*!
+ *  Compile an onnx model from a file containing MLIR or ONNX protobuf
+ *  @param inputFilename File name pointing onnx model protobuf or MLIR
+ *  @param outputBaseName File name without extension to write output
+ *  @param emissionTarget Target format to compile to
+ *  @param mcpu Optional target CPU
+ *  @param mtriple Optional target architecture
+ *  @return 0 on success or non-zero error code on failure
+ */
+ONNX_MLIR_EXPORT int CompileFromFile(const char *inputFilename,
```

Comment:
All of the names in the ONNX MLIR runtime are prefaced with "om". I suggest we do the same here.